### PR TITLE
remove cors package completely and utilise the domain whitelist

### PR DESCRIPTION
**Pull request by Baalmart**
**_WHAT DOES THIS PR DO?_**
remove cors package completely and utilise the domain whitelist.

**_ACTIONS REQUIRED FOR DEPLOYMENT?_**
@123MwanjeMike 
Update the domain WHITELIST for both _staging_ and _production_. Current one for _staging_ is:
`DOMAIN_WHITELIST: https://staging.airqo.net/,https://airqo.net/,https://airqo.africa/,https://airqo.org/,https://airqo.mak.ac.ug/,https://airqo.io/,https://platform.airqo.net/,https://staging-platform.airqo.net/,http://127.0.0.1,http://localhost`

Reference: https://stackoverflow.com/questions/24897801/enable-access-control-allow-origin-for-multiple-domains-in-node-js

Screenshot 2022-09-09 at 11 46 57

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - [AN-136]

[AN-136]: https://airqoteam.atlassian.net/browse/AN-136?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/device-registry/routes/api-v1.js [hunk 1]
```diff
@@ -18,20 +18,13 @@ const { isBoolean, isEmpty } = require("underscore");
 const phoneUtil = require("google-libphonenumber").PhoneNumberUtil.getInstance();
 const decimalPlaces = require("decimal-places");
 const activityController = require("../controllers/create-activity");
-const cors = require("cors");
-
-const corsOptionsDelegate = function(req, callback) {
```
Comment (Baalmart) [inline src/device-registry/routes/api-v1.js:23]: removing the utilisation of the` cors` package

File: src/device-registry/package.json [hunk 2]
```diff
@@ -29,7 +29,6 @@
         "compression": "^1.7.4",
         "connect-mongo": "^3.2.0",
         "cookie-parser": "~1.4.3",
-        "cors": "^2.8.5",
```
Comment (Baalmart) [inline src/device-registry/package.json:32]: removing the utilisation of the` cors` package

**Review by Baalmart — COMMENTED** [review]
Hi @123MwanjeMike , please check out the README of this PR accordingly. It has some action items we need to take before we merge and test accordingly. Thanks!

**Comment by 123MwanjeMike** [comment]
> Hi @123MwanjeMike , please check out the README of this PR accordingly. It has some action items we need to take before we merge and test accordingly. Thanks!

Hi @Baalmart 
Please note that the value has been updated to that below
`https://staging.airqo.net/,https://airqo.net/,https://airqo.africa/,https://airqo.org/,https://airqo.mak.ac.ug/,https://airqo.io/,https://platform.airqo.net/,https://staging-platform.airqo.net/,http://127.0.0.1,http://localhost,http://127.0.0.1:5000,http://localhost:5000`

**Review by 123MwanjeMike — APPROVED** [review]
Thanks @Baalmart 
LGTM

**Pull request merged**